### PR TITLE
Update Twig to v1.38

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update Twig to v1.38 to take care of https://symfony.com/blog/twig-sandbox-information-disclosure
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No view using Twig is broken 😶

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13012)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]
Because of security issue please update composer.json
```
        "twig/twig": "^1.38||^2.0"
```

File: composer.json [hunk 1]
```diff
@@ -108,10 +108,7 @@
         "symfony/swiftmailer-bundle": "^2.6.4",
         "symfony/symfony": ">=3.4.8",
         "tecnickcom/tcpdf": "^6.2.12",
-        "twig/twig": "^1.0||^2.0"
-    },
-    "conflict": {
-        "twig/twig": "1.37.0|2.6.1"
+        "twig/twig": "^1.38||^2.0"
```
Comment (PierreRambaud) [inline composer.json]:
```suggestion
        "twig/twig": "^1.38||^2.6.2"
```

**Comment by PierreRambaud** [comment]
Fix conflicts and merge? :)

**Comment by Quetzacoalt91** [comment]
Rebased and removed `composer.lock` from the diff, as we already upgraded to 1.38.4.

**Review by Quetzacoalt91 — DISMISSED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**